propagators/aws/xray/sampled_unsampled_span_test.go

```diff
@@ -0,0 +1,104 @@
+// Copyright The OpenTelemetry Authors
```

Comment:
This file should be renamed to something descriptive of the benchmarks included. I.e. `idgenerator_benchmark_test.go`